napari/layers/_vispy/util.py

```diff
@@ -0,0 +1,29 @@
+from .vispy_base_layer import VispyBaseLayer
+from .vispy_image_layer import VispyImageLayer
+from .vispy_labels_layer import VispyLabelsLayer
+from .vispy_vectors_layer import VispyVectorsLayer
+from .vispy_volume_layer import VispyVolumeLayer
+
+
+def create_vispy_node(layer):
+    """
+    Create vispy visual node for a layer based on its layer type.
+
+    Parameters
+    ----------
+        layer : napari.layers._base_layer.Layer
+            Layer that needs its propetry widget created.
+
+    Returns
+    ----------
+        node : vispy.scene.visuals.VisualNode
+            Vispy visual node
+    """
+    name = 'Vispy' + type(layer).__name__ + 'Layer'
+    try:
+        VispyLayer = globals()[name]
```

Comment:
This seems brittle...? What's happening here? I think you're accessing the module namespace... I suggest instead to create a dictionary mapping names to objects.

Reply:
This remains unaddressed... =)

Reply:
replaced with a dictionary - both here and in the `_qt/utils.py` where a similar pattern appeared - let me know if that's what you had in mind